### ISSUE TITLE
Does not work with portable Windhawk installation

### ISSUE THREAD

**Issue by jacklul** [ramensoftware/windows-11-start-menu-styling-guide#90]
Seems like this mod refuses to work when Windhawk is running as portable installation.

**Comment by m417z** [comment]
It was a known limitation with earlier portable versions. The reason was that the start menu process runs in an app container with limited access to files, and it couldn't access Windhawk files. The installer of newer Windhawk versions sets adjusts file permissions in such a way that it should work. So if you have an old Windhawk version or if you moved the files around, and even if not, you might want to just run the latest installer of Windhawk again to override the files and set the permissions again. If that doesn't help, let me know and I'll investigate further.

**Comment by jacklul** [comment]
Yes, I did move the file after the installer unpacked it, any reasons why would the permissions not stick?

**Comment by m417z** [comment]
If you moved it to a location such as a disk-on-key, perhaps the file system doesn't support such permissions. But generally no idea.
